docs/content/docs/tutorials/memorydb-example.md

```diff
@@ -0,0 +1,288 @@
+MEMORYDB_CLUSTER_CONN_CM="${MEMORYDB_CLUSTER_NAME}-conn-cm"
+
+cat <<EOF > memorydb-field-exports.yaml
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${MEMORYDB_CLUSTER_CONN_CM}
+data: {}
+---
+apiVersion: services.k8s.aws/v1alpha1
+kind: FieldExport
+metadata:
+  name: ${MEMORYDB_CLUSTER_NAME}-host
+spec:
+  to:
+    name: ${MEMORYDB_CLUSTER_CONN_CM}
+    kind: configmap
+  from:
+    path: "status.ClusterEndpoint.Address"
+    resource:
+      group: memorydb.services.k8s.aws
+      kind: Cluster
+      name: ${MEMORYDB_CLUSTER_NAME}
+---
+apiVersion: services.k8s.aws/v1alpha1
+kind: FieldExport
+metadata:
+  name: ${MEMORYDB_CLUSTER_NAME}-port
+spec:
+  to:
+    name: ${MEMORYDB_CLUSTER_CONN_CM}
+    kind: configmap
+  from:
+    path: "status.ClusterEndpoint.Port"
+    resource:
+      group: memorydb.services.k8s.aws
+      kind: Cluster
+      name: ${MEMORYDB_CLUSTER_NAME}
+EOF
+
+kubectl apply -f memorydb-field-exports.yaml
+```
+
+These values can be injected into a container either as environmental variables or files. For example, here is a snippet of a Deployment definition that will add the MemoryDB Cluster connection info into the Pod:
+```shell
+cat <<EOF > game_leaderboard.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: leaderboard-deployment
+  labels:
+    app: leaderboard
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: leaderboard
+  template:
+    metadata:
+      labels:
+        app: leaderboard
+    spec:
+      containers:
+      - name: leaderboard
+        image: public.ecr.aws/nginx/nginx:latest
+        ports:
+          - containerPort: 80
+        env:
+          - name: MEMORYDB_CLUSTER_HOST
+            valueFrom:
+              configMapKeyRef:
+                name: ${MEMORYDB_CLUSTER_CONN_CM}
+                key: "${MEMORYDB_CLUSTER_NAME}-host"
+          - name: MEMORYDB_CLUSTER_PORT
+            valueFrom:
+              configMapKeyRef:
+                name: ${MEMORYDB_CLUSTER_CONN_CM}
+                key: "${MEMORYDB_CLUSTER_NAME}-port"
+EOF
+
+kubectl apply -f game_leaderboard.yaml
+```
+
+## Next steps
+
+You can learn more about each of the ACK service controller for MemoryDB custom resources by using `kubectl explain` on the API resources. These include:
+
+* `cluster`
+* `user`
+* `acl`
+* `parametergroup`
+* `subnetgroup`
+
+The ACK service controller for Amazon MemoryDB is based on the [Amazon MemoryDB API](https://docs.aws.amazon.com/memorydb/latest/APIReference/Welcome.html). To get a full understanding of how all of the APIs work, please review the [Amazon MemoryDB API documentation](https://docs.aws.amazon.com/memorydb/latest/APIReference/Welcome.html).
+
+You can learn more about [how to use Amazon MemoryDB](https://docs.aws.amazon.com/memorydb/index.html) through the [documentation](https://docs.aws.amazon.com/memorydb/index.html).
+
+### Cleanup
+
+You can deprovision your MemoryDB cluster using `kubectl delete` command.
```

Comment:
And the rest of the resources?

Reply:
Updated the document to delete the example resources that the tutorial creates.